cmd2/table_creator.py

```diff
@@ -141,8 +136,20 @@ def __init__(self, cols: Sequence[Column], *, tab_width: int = 4) -> None:
         self.cols = copy.copy(cols)
         self.tab_width = tab_width
 
+        for col in self.cols:
+            # Replace tabs before calculating width of header strings
+            col.header = col.header.replace('\t', SPACE * self.tab_width)
+
+            # For headers with the width not yet set, use the width of the
+            # widest line in the header or 1 if the header has no width
+            if col.width is None:
+                line_widths = [ansi.style_aware_wcswidth(line) for line in col.header.splitlines()]
+                line_widths.append(1)
+                col.width = max(line_widths)
+
     @staticmethod
-    def _wrap_long_word(word: str, max_width: int, max_lines: Union[int, float], is_last_word: bool) -> Tuple[str, int, int]:
+    def _wrap_long_word(word: str, max_width: int, max_lines: Union[int, float],
```

Comment:
I haven't tested it yet, but I read recently that instead of `Union` you can use syntax like the folliwing:
```Python
def _wrap_long_word(word: str, max_width: int, max_lines: int | float):
```

I'm not sure if it requires a newer version of Python or not, but sometime we should try it out - I think it is a lot cleaner and more readable than using `Union[int, float]` and is also easier when people are moving between Python and TypeScript.